llvm/docs/GettingStarted.rst

```diff
@@ -1085,67 +1085,88 @@ general questions about LLVM, please consult the `Frequently Asked
 Questions <FAQ.html>`_ page.
 
 If you are having problems with limited memory and build time, please try
-building with ninja instead of make. Please consider configuring the
+building with ``ninja`` instead of ``make``. Please consider configuring the
 following options with cmake:
 
- * -G Ninja
+ * ``-G Ninja``
+
    Setting this option will allow you to build with ninja instead of make.
    Building with ninja significantly improves your build time, especially with
    incremental builds, and improves your memory usage.
 
- * -DLLVM_USE_LINKER
-   Setting this option to lld will significantly reduce linking time for LLVM
+ * ``-DLLVM_USE_LINKER``
+
+   Setting this option to ``lld`` will significantly reduce linking time for LLVM
    executables on ELF-based platforms, such as Linux. If you are building LLVM
```

Comment:
This is also true on Windows, not really on macOS any longer with the introduction of ld-prime, but it's just not linux.

Reply:
I've mentioned Linux and Windows, vaguely enough that we don't have to make any statements of quality about the Mac linker, or the many *BSDs where lld is also an improvement.